src/solver/conv_direct_naive_conv_bwd.cpp

```diff
@@ -33,6 +33,9 @@ MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_CONV_DIRECT_NAIVE_CONV_BWD)
 
 namespace miopen {
 namespace solver {
+namespace conv {
```

Comment:
I'd like to know what the benefits of introducing this namespace `solver::conv` are over not introducing it, because it seems that currently it's requiring developers to disambiguate between `miopne::conv` and `miopen::solver::conv` in their code and that seems to add diffculty.

Reply:
We already have `solver::fusion`, `solver::activ`, `solver::pooling` and `solver::batchnorm`, and there are no problems with them. But we don't have `solver::conv` for some reason, and I think if we add it the code will be more uniform and will be structured better